src/commands/application/instances/list.rs

```diff
@@ -67,15 +67,18 @@ pub async fn handle_list(
     let instances: Vec<Instance> = k8s_pods
         .into_iter()
         .map(|pod| Instance {
-            name: pod.name,
+            name: format!("{}@{}/{}", version, namespace, pod.name),
             ip: pod.host_ip.unwrap_or_default(),
             ready: pod.ready,
         })
         .collect();
 
     fetching_progress_bar.finish_and_clear();
 
-    eprintln!("Listing running instances of the application mv-wukong-ci-mock on namespace production...✅");
+    eprintln!(
+        "Listing running instances of the application {} on namespace production...✅",
```

Comment:
since we're listing on all namespaces, i think the text `on namespace production` is redundant. We should remove it.

Reply:
got it